docs/content/ecs/entity-contracts.md

```diff
@@ -44,19 +43,19 @@ The following guarantees hold for any ECS Entity representing a SpatialOS entity
 ### Guarantees when receiving updates or messages
 Whenever a component update or message is received by the worker, the following holds for the entity that the worker received it for:
 
-  * All updates received are stored as a list in a reactive component called `ReceivedUpdates<T>`, where `T` is the type of the component.
+  * All updates received are stored as a list in a reactive component called `{name of component}.ReceivedUpdates`.
```

Comment:
do we only have the reactive version of that for ecs?

Reply:
There's an API on the `ComponentUpdateSystem` to get all updates of a type `T`. 

Not documenting these APIs just yet

Reply:
fair enough, do we have a ticket or todo tracking that?

Reply:
Yes - https://improbableio.atlassian.net/browse/UTY-1683